dot-net-sdk/EppoClient.cs

```diff
@@ -15,6 +16,7 @@ namespace eppo_sdk;
 
 public class EppoClient
 {
+    private const string AssignmentDefaultValue = "__DEFAULT__";
```

Comment:
sentinel default value to note when a string assignment is not made for flag/subject/attributes